jina/drivers/rank.py

```diff
@@ -148,3 +148,55 @@ def _apply_all(self, docs: Iterable['jina_pb2.Document'], context_doc: 'jina_pb2
                 r.score.ref_id = context_doc.id  # label the score is computed against doc
                 r.score.value = score
                 r.score.op_name = exec.__class__.__name__
+
+
+class Matches2DocRankDriver(BaseRankDriver):
```

Comment:
I like this naming, but for now to highlight the difference of `Chunk2DocRankerDriver` I may name it `Doc2DocRankerDriver`? In a future PR I would totally refactor names, what do you think?

Reply:
But this ranker explicitly only goes from matches to doc. So why make the naming more general then it is?